CITi-UFPE/CITiMarco
# Feature/when input part 1

## What I did

- Create Date Ranger Picker

## How to test

1. Run the project, on http://localhost:3000, you should see the basic structure for the date picker.
- Disclaimer: when you change the initial date to something after the final date, a big error shows up and breaks things, I will fix this in another PR. And I will still change the styles when the mockup is done.
